youtube_dl/extractor/svt.py

```diff
@@ -320,17 +320,24 @@ def _real_extract(self, url):
 class SVTPageIE(InfoExtractor):
     _VALID_URL = r'https?://(?:www\.)?svt\.se/(?:[^/]+/)*(?P<id>[^/?&#]+)'
     _TESTS = [{
-        'url': 'https://www.svt.se/sport/oseedat/guide-sommartraningen-du-kan-gora-var-och-nar-du-vill',
+        'url': 'https://www.svt.se/nyheter/vetenskap/har-ar-klimatrapporten-i-fem-punkter',
         'info_dict': {
-            'id': 'guide-sommartraningen-du-kan-gora-var-och-nar-du-vill',
-            'title': 'GUIDE: Sommarträning du kan göra var och när du vill',
+            'id': 'har-ar-klimatrapporten-i-fem-punkter',
+            'title': 'Här är FN:s klimatrapport i fem punkter',
         },
-        'playlist_count': 7,
+        'playlist_count': 1,
+    }, {
+        'url': 'https://www.svt.se/nyheter/utrikes/svenska-andrea-ar-en-mil-fran-branderna-i-kalifornien',
+        'info_dict': {
+            'id': 'svenska-andrea-ar-en-mil-fran-branderna-i-kalifornien',
+            'title': 'Svenska Andrea redo att fly sitt hem i Kalifornien',
+        },
+        'playlist_count': 2,
     }, {
-        'url': 'https://www.svt.se/nyheter/inrikes/ebba-busch-thor-kd-har-delvis-ratt-om-no-go-zoner',
+        'url': 'https://www.svt.se/nyheter/lokalt/norrbotten/komplicerat-att-stanga-mr-kameran',
         'info_dict': {
-            'id': 'ebba-busch-thor-kd-har-delvis-ratt-om-no-go-zoner',
-            'title': 'Ebba Busch Thor har bara delvis rätt om ”no-go-zoner”',
+            'id': 'komplicerat-att-stanga-mr-kameran',
+            'title': 'MR-kameran fortfarande igång – komplicerad avstängning påbörjad',
```

Comment:
Remove duplicate tests.

Reply:
I'm sorry I don't exactly see what you mean. The tests in `SVTPageIE` are all unique.